tests/FSharpLint.Core.Tests/Rules/Conventions/SourceLength.fs

```diff
@@ -4,7 +4,13 @@ open NUnit.Framework
 open FSharpLint.Rules
 open FSharpLint.Rules.Helper.SourceLength
 
-let generateNewLines numNewLines = Array.create numNewLines "\n" |> String.concat ""
+let generateNewLines numNewLines numIndents =
+    Array.create numNewLines "\n"
+    |> String.concat (sprintf "%sprintf \"\"" (String.replicate numIndents " "))
+
+let generateAbstractMembers numMembers numIndents =
+    Array.init numMembers (fun index -> sprintf "abstract member Foo%d : unit -> unit\n" index)
```

Comment:
@Mersho our F# guidelines say you have to use %i for integers, why do you keep using d if i->integer is easier to remember? what does d mean to you?

Reply:
I think d stands for digit, it's a habit from other programming languages (such as CPP).